consensus/istanbul/backend/backend.go

```diff
@@ -149,20 +149,10 @@ func New(config *istanbul.Config, db ethdb.Database) consensus.Istanbul {
 				"cycle", "sleep", "consensus", "block_verify", "block_construct",
 				"sysload", "syswait", "procload")
 		}
-
 	}
 
 	backend.core = istanbulCore.New(backend, backend.config)
 
-	backend.logger = istanbul.NewIstLogger(
-		func() *big.Int {
-			if backend.core != nil && backend.core.CurrentView() != nil {
-				return backend.core.CurrentView().Round
-			}
-			return common.Big0
-		},
-	)
-
```

Comment:
This is going to change the logging functionality, previously when this was in use if round 2 was reached then Trace and Debug messages would be output at the Info level. Now this is not the case.

The original PR that introduced the IstLogger doesn't provide any explanation of why the change was needed.

I would actually be in favour of removing the IstLogger altogether since it seems like the kind of thing that would confuse node operators (why did my logging output jut increase massively and why are the logging levels different from what I'm used to seeing?)

@oneeman @mcortesi would you be able to provide any insight here?

Reply:
I'm not familiar with it.  My guess would be that the idea was that if there's a round change it may indicate things going wrong (though it can also happen due to the proposer being offline) and that you'd want more logging in that case.  But I don't know whether it has proven to be useful in the end, and at least I wasn't even aware of it.

Reply:
that's the theory. Another nice part of the istLogger is the auto population of the round state information (round, sequence, desiredRound, state, etc)

Reply:
I spoke with @timmoreton and yeah the theory was that it'd be useful to have the extra logging if there was a round change because something went wrong. But this can also be achieved on a per validator basis by increasing the log verbosity through the console using the debug api and given that he didn't see any reason this couldn't be removed.

 So I'm in favour of removing it because that fixes this immediate problem, and puts control back in the hands of validators, allowing them to increase the logging level if they need it but not forcing it upon them.

Reply:
@trianglesphere Could you point me to the "auto population of the round state information" ? I can't see anything relating to that in the ist logger code.

Reply:
@piersy It's actually `(c *core) newLogger(ctx ...interface{}) log.Logger` which does the auto-population. My bad as I got confused between that and the istLogger.